src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -62,17 +82,36 @@ def delete_configstore(cmd, client, name, resource_group_name=None, yes=False):
     return client.begin_delete(resource_group_name, name)
 
 
+def purge_deleted_configstore(cmd, client, name, location=None, yes=False):
+    if location is None:
+        _, location = resolve_deleted_store_metadata(cmd, name)
+    confirmation_message = "This operation will permanently delete App Configuration and it's contents.\nAre you sure you want to purge the App Configuration: {}".format(name)
+    user_confirmation(confirmation_message, yes)
+    return client.begin_purge_deleted(config_store_name=name, location=location)
+
+
 def list_configstore(client, resource_group_name=None):
     response = client.list() if resource_group_name is None else client.list_by_resource_group(resource_group_name)
     return response
 
 
+def list_deleted_configstore(client):
+    response = client.list_deleted()
+    return response
```

Comment:
>response [](http://example.com/codeflow?start=11&length=8)

Does it also work for other output formats like table? If there are some new attributes we want to show as column name, we will need to change in _format.py.

Reply:
Good catch. I didn't even know there was a table format.
I have now added a table formatter for deleted stores.